Formula/gpg-tui.rb

```diff
@@ -0,0 +1,33 @@
+class GpgTui < Formula
+  desc "Manage your GnuPG keys with ease! 🔐"
+  homepage "https://github.com/orhun/gpg-tui"
+  url "https://github.com/orhun/gpg-tui/archive/refs/tags/v0.1.5.tar.gz"
+  sha256 "eccfaa3089605bb6df2cc49df208a93cc452d8a819b444bf82dfeb90739135ea"
+  license "MIT"
+
+  depends_on "rust" => :build
+  depends_on "gnupg"
+  depends_on "gpgme"
+  depends_on "libgpg-error"
+  depends_on "libxcb"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    completions = share/"completions"
+    mkdir_p completions
+    ENV["OUT_DIR"] = completions
+    system bin/"completions"
+    bash_completion.install completions/"gpg-tui.bash"
+    fish_completion.install completions/"gpg-tui.fish"
+    zsh_completion.install completions/"_gpg-tui"
+
+    rm_rf completions
+    rm_f bin/"completions"
+    rm_f Dir[prefix/".crates*"]
+  end
+
+  test do
+    assert_match "gpg-tui [FLAGS] [OPTIONS]", shell_output("#{bin}/gpg-tui --help 2>&1")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (`foo --version` or `foo --help`) are not sufficient, as explained in the formula cookbook. In most cases, a good test would involve running a simple test case and/or comparing some output.

Reply:
Unfortunately, this TUI is (at least currently) fully interactive and does not allow meaningful CLI-only actions except `--help` and `--version`.

Reply:
(Carlo's suggestion below provides more context so I'm resolving this.)